application/lib/tutorial/six/module/chatModel.dart

```diff
@@ -0,0 +1,9 @@
+class ChatModel {
+  int id;
+  String message;
+  bool isMe;
```

Comment:
What's `isMe`?

Reply:
I originally thought we need to reply to the user in the chat room (challenge module) so added isMe to determine who's sending a message. (True == user)